test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,12 +25,13 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 0 IPv4 -mutexTimeStat
+brokerStart CB 0 IPv4 -mutexTimeStat -subCacheIval 0
```

Comment:
Not sure how to respond on this ...
Timing issues, definitely, but ... how come we're not able to fix the problem in the atexit-function?
Frankly, I didn't have time enough to fix everything ...
This potential leak is only about when the broker dies. Not especially interesting.
An issue about removing this option is definitely in order though.  @fgalan ? :-)